ReactAndroid/src/main/java/com/facebook/react/ReactActivityDelegate.java

```diff
@@ -29,6 +30,10 @@
  * class doesn't implement {@link ReactApplication}.
  */
 public class ReactActivityDelegate {
+
+  private final int REQUEST_OVERLAY_CODE = 1111;
```

Comment:
>  It doesn't need to be unique to other application just your own

It can conflict with third party modules / and even own code. Since this code is part of the RN library, not my application.